src/pres/loader/Filters.js

```diff
@@ -23,9 +23,12 @@ export default class User extends React.Component {
 
     constructor(props) {
         super(props)
+
+        let isAdvancedFiltersOpen = new URLSearchParams(window.location.search).get("advanced") === 'true'
+
```

Comment:
Also, arguably we should do all the "looking at params" stuff in one place (not spread out across multiple components like this)